docs/api-references/user.md

```diff
@@ -0,0 +1,127 @@
+## User API
+
+### `GET` User
+```
+/api/user/
+```
+Get information about a current user login.
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
+```
+#### Sample Response
+```json
+{
+    "data": {
+                "id": 74,
+                "full_name": "dungvan2512",
+                "email": "dungvan@test.email.com",
+                "birthday": null,
+                "gender": 0,
+                "address": null,
+                "phone_number": null,
+                "image": "http://foodmarket.com/images/users/default.jpg",
+                "is_admin": 1,
+                "is_active": 0,
+                "created_at": "2017-08-23 04:13:40",
+                "updated_at": "2017-08-23 04:13:40",
+                "deleted_at": null
+            },
+    "success": true
+}
+```
+
+### `Post` user
+```
+/api/user/create
+```
+Registry a food
+
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| email | Email | required | email to login |
+| password | Password | required, least at 6 character | password |
+| password_confirmation | Password | required, map to password | password confirmation |
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
+```
+
+#### Sample Request body
+```json
+{
+   "full_name": "Van Duc Dung",
+   "email": "vandung@test.email.com",
+   "password": "123456",
+   "password_confirmation": "123456"
+}
+```
+
+#### Sample Response
+```json
+{
+    "data": {
+        "id": 74,
+                        "full_name": "Van Duc Dung",
+                        "email": "vandung@test.email.com",
+                        "birthday": null,
+                        "gender": 0,
+                        "address": null,
+                        "phone_number": null,
+                        "image": "http://foodmarket.com/images/users/default.jpg",
+                        "is_admin": 0,
+                        "is_active": 0,
+                        "created_at": "2017-08-23 04:13:40",
+                        "updated_at": "2017-08-23 04:13:40",
+                        "deleted_at": null
+    },
+    "success": true
+}
+```
+### `Put` user
+```
+/api/user/edit
+```
+Registry a food
+
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| full_name | String | required | Name of user |
+| birthday | Date | required | email to login |
+| gender | Integer | required | Default 0 : female, 1 : male  |
+| address | String | required | address of user |
+| phone_number | String | required, number | phone number of user |
+| password | Password | required, least at 6 character | password |
+| password_confirmation | Password | nullable, map to password | password confirmation |
+
+#### Request header
+```
+  "Content-Type": "application/json",
+  "Accept": "application/json",
```

Comment:
`"Accept": "application/json",` is OK, but why only this API has `Accept`. Please make unification for all APIs.